migrations/2021-06-21-093252_create_games/up.sql

```diff
@@ -0,0 +1,29 @@
+CREATE TABLE games (
+    id INTEGER PRIMARY KEY
+);
+
+-- Going to create a separate table for players to avoid player_1_id, player_2_id etc columns in future.
+CREATE TABLE players (
```

Comment:
and players is a bit of an odd name for this -- I think this should be `games`, and the id pk will be the "game_id"

Reply:
So the combination of game_id and user_id is the pk?